src/Sylius/Behat/Context/Ui/CartContext.php

```diff
@@ -46,6 +46,7 @@ public function __construct(
 
     /**
      * @Given I added product :product to the cart
+     * @Given I had product :product in the cart
      * @When I add product :product to the cart
      * @When I have product :product in the cart
```

Comment:
It used in [several steps](https://github.com/Sylius/Sylius/blob/master/features/shipping/apply_correct_shipping_fee_on_order.feature#L16), where having a product in cart should be presumption, not an action performed.